### PR TITLE
Fix wizard browse step

### PR DESCRIPTION
**Pull request by plata**
fixes https://github.com/PhoenicisOrg/scripts/issues/1054

### CODE REVIEW

**Comment by madoar** [comment]
When looking at our Java - JavaScript interop code I discover that we have almost no input/output validation (i.e. is `directory` a valid directory?, is the result for a browse operation valid?). 

I think we should add such checks to the code.

File: phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/UiSetupWizardImplementation.java [hunk 1]
```diff
@@ -283,8 +283,9 @@ public String browse(String textToShow) {
      */
     @Override
     public String browse(String textToShow, String directory, List<String> allowedExtensions) {
+        final List<String> copiedAllowedExtensions = allowedExtensions != null ? List.copyOf(allowedExtensions) : null;
```
Comment (madoar) [inline phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/UiSetupWizardImplementation.java]: Are you sure it should be null and not an empty List?

Comment (plata) [reply phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/UiSetupWizardImplementation.java]: It's the same behavior like before.

**Comment by plata** [comment]
Please open a new issue for the particular input/output validation you would like to have.

**Comment by madoar** [comment]
See #2032

**Pull request merged**